webknossos/webknossos/annotation/annotation.py

```diff
@@ -92,24 +93,54 @@ def binary(self) -> Union[bytes, BinaryIO]:
         else:
             return open(self.file, "rb")
 
+    def get_volume_layer_names(self) -> Iterable[str]:
+
+        return (volume._get_name_or_id() for volume in self._nml.volumes)
+
     def save_volume_annotation(
         self,
         dataset: Dataset,
         layer_name: str = "volume_annotation",
         largest_segment_id: Optional[int] = None,
+        source_volume_name: Optional[str] = None,
     ) -> Layer:
         """
         Given a dataset, this method will save the
         volume annotation of this annotation into that dataset
         by creating a new layer.
         The largest_segment_id is computed automatically, unless provided
         explicitly.
+
+        source_volume_name has to be provided, if the annotation contains
+        multiple volume layers. Use Annotation.get_volume_layer_names() to look up
+        available layers.
```

Comment:
```suggestion
        `source_volume_name` has to be provided, if the annotation contains
        multiple volume layers. Use `get_volume_layer_names()` to look up
        available layers.
```
(the method `get_volume_layer_names()` will be auto-linked, since it's within the same module